browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -77,7 +77,7 @@ void EphemeralStorageTabHelper::ReadyToCommitNavigation(
   if (new_domain == previous_domain)
     return;
 
-  CreateEphemeralStorageAreasForDomainAndURL(new_domain, new_url);
+  CreateEphemeralStorageAreasForDomainAndURL(std::move(new_domain), new_url);
```

Comment:
wouldn't it make more sense to change the param to `const std::string&`. Not sure why it wasn't a const ref in the first place